backend/tracim_backend/lib/mail_notifier/sender.py

```diff
@@ -71,6 +71,7 @@ def connect(self):
             if self._smtp_config.login:
                 try:
                     starttls_result = self._smtp_connection.starttls()
+                    logger.debug(self, 'SMTP Starttls OK')
```

Comment:
you dont know here if its ok: you didnt check. By the way, I believe that its better to write `Start TLS`

Reply:
@inkhey the issue is to log information that has been checked.

Example: if you write "start tls failed", the user understand that you have something like "if starttls is error: blablabla else bliblibli". In the code you wrote, you check nothing.